src/lib/reasoners/adt_rel.ml

```diff
@@ -319,45 +319,9 @@ let empty uf = {
 (* ################################################################ *)
 (*BISECT-IGNORE-BEGIN*)
 module Debug = struct
-  open Printer
-
-  let assume a =
-    if Options.get_debug_adt () then
-      print_dbg
-        ~module_name:"Adt_rel"
-        ~function_name:"assume"
-        "assume %a" LR.print (LR.make a)
-
-  let pp_domains loc domains =
-    if Options.get_debug_adt () then begin
-      print_dbg ~flushed:false ~module_name:"Adt_rel"
-        "@[<v 2>--ADT env %s ---------------------------------@ " loc;
-      print_dbg ~flushed:false ~header:false "domains:@ %a"
-        Domains.pp domains;
-      print_dbg ~header:false "---------------------"
-    end
-
-  (* unused --
-     let case_split r r' =
-     if get_debug_adt () then
-       Printer.print_dbg
-          "[ADT.case-split] %a = %a" X.print r X.print r'
-  *)
-
-  let no_case_split () =
-    if Options.get_debug_adt () then
-      print_dbg
-        ~module_name:"Adt_rel"
-        ~function_name:"case-split"
-        "nothing"
-
-  let add r =
-    if Options.get_debug_adt () then
-      print_dbg
-        ~module_name:"Adt_rel"
-        ~function_name:"add"
-        "%a" X.print r
-
+  let assume l =
+    Logs.debug ~src:Sources.adt_rel
```

Comment:
Please don't use `Logs.debug ~src:...`. Instead, define `module Log = (val Logs.src_log Sources.adt_rel)` at the top of the file, and use `Log.debug` directly. See the [example](https://erratique.ch/software/logs/doc/Logs/index.html#setupreporter) from `Logs`'s documentation.